src/main/java/org/store/book/Book.java

```diff
@@ -0,0 +1,90 @@
+package org.store.book;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+public class Book {
```

Comment:
Remove unneccessary setter methods for members that would not change once initialized.